.dev/format.sh

```diff
@@ -8,7 +8,7 @@
 
 format() {
     # don't use a directory with whitespace
-    local whitelist="apps/3d_rec_framework apps/modeler 2d geometry ml octree simulation stereo tracking registration gpu/containers"
+    local whitelist="apps/3d_rec_framework apps/include apps/modeler apps/src 2d geometry ml octree simulation stereo tracking registration gpu/containers"
```

Comment:
Can we use just apps now?

Reply:
Nop, cloud_composer, in_hand_scanner & modeler are still missing